src/emm/emm_cli.py

```diff
@@ -211,5 +225,25 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100))
+@click.argument("revision", nargs=-1)
+@click.option(
+    "-o",
+    "--operation",
+    type=click.Choice([a.value for a in GitAction]),
+    default=GitAction.CHECKOUT,
+    help="Git operations to perform with found commit [default=checkout].",
+)
+@click.option("-b", "--branch", default=None, help="Name of branch for git checkout.")
+@click.option("--directory", default=None, help="Directory to execute command at.")
```

Comment:
Same comment as in the other PR, I don't think we want to accept a directory for these commands.